app/categories/more_problems/constants.py

```diff
@@ -0,0 +1,97 @@
+from app.categories.constants import Category
+from flask_babel import lazy_gettext as _
+
+ADOPTING = Category(
```

Comment:
Can you add these categories to app/categories/constants.py. Also they should all have a unique code